app/styles/_core.less

```diff
@@ -466,41 +466,77 @@ label.checkbox {
 
 .osc-secrets-form {
   .basic-secrets {
```

Comment:
You could significantly DRY up this LESS like so:

```LESS
.osc-secrets-form {
  .advanced-secrets,
  .basic-secrets {
    .input-labels,
    .help-blocks {
      display: table;
      padding-right: 30px;
      position: relative;
      width: 100%;
      .input-label,
      .help-block {
        width: 100%;
        float: left;
        padding-right: 5px;
      }
    }
    .secret-row {
      display: table;
      padding-right: 30px;
      position: relative;
      width: 100%;
      padding-bottom: 5px;
      .secret-name {
        float: left;
        padding-right: 5px;
        width: 100%;
      }
      .remove-secret {
        position: absolute;
        right: 0;
        top: 0;
        width: 30px;
      }
    }
  }

  .advanced-secrets {
    .input-labels,
    .help-blocks {
      .input-label,
      .help-block {
        width: 50%;
      }
    }
    .secret-row {
      .secret-name,
      .destination-dir {
        width: 50%;
      }
    }
  }

  .remove-btn {
    color: #333;
    margin-left: 5px;
    opacity: 0.65;
    font-size: 15px;
    vertical-align: middle;
    &:hover {
      opacity: 1;
      text-decoration: none;
    }
    &:focus {
      text-decoration: none;
    }
  }
}
```

Note:  this is untested, so make sure you double check.